### ISSUE TITLE
Bug: Inital dock icon too small

### ISSUE THREAD

**Issue by limstash** [Foundry376/Mailspring#1963]
**Describe the bug**
The size of inital dock icon is abnormal. It's too small to notice. 

**To Reproduce**
1. Just start the software

**Expected behavior**
![db8b9dd15993a0d056123b76cd6ae58e.png](https://img.yellowfish.top/images/2020/05/03/db8b9dd15993a0d056123b76cd6ae58e.png)

Uncheck the [Preference - Show icon in menu bar / system tray] and then check it, it turns normal.
 
**Screenshots**
![e07e627ba8c32abdda64e223a6e02556.png](https://img.yellowfish.top/images/2020/05/03/e07e627ba8c32abdda64e223a6e02556.png)

**Setup: (please complete the following information):**
 - OS: Archlinux (Deepin Desktop Environment)
 - OS Version: rolling
 - Mailspring Version: 1.7.6

**Comment by bengotow** [comment]
Hey thanks for reporting this! That's no good at all, and it's really interesting that it works after a reset.

Also weird that it looks like it's incredibly low resolution compared to your other icons (which I suspect are vector graphics?)

Do you know where I could get a VMWare VM for an Archlinux install similar to yours? I haven't been able to test on arch and it'd be nice to fire it up and dig in to this a bit myself.

**Comment by limstash** [comment]
### Step 1

You could get archlinux iso file from [this link](http://mirror.rackspace.com/archlinux/iso/2020.05.01/archlinux-2020.05.01-x86_64.iso)

Attach this iso file in the VM (disable secure boot)

### Step 2
Switch to Arch Linux archiso x86_64 UEFI CD

Enable NTP
```bash
timedatectl set-ntp true
```

use `fdisk` to create edit partition table

```bash
fdisk /dev/sda
```

create 3 partitions
- `/dev/sda1` (fat32) for UEFI (need boot flag)
- `/dev/sda2` (ext4) for `/boot`
- `/dev/sda3` (ext4) for `/`

```bash
mkfs.fat -F 32 /dev/sda1
mkfs.ext4 /dev/sda2
mkfs.ext4 /dev/sda3
```

### Step 3
```bash
mount /dev/sda3 /mnt
mkdir /mnt/boot
mount /dev/sda2 /mnt/boot
mkdir /mnt/boot/EFI
mount /dev/sda1 /mnt/boot/EFI
```

Install archlinux

```bash
pacstrap -i /mnt base base-devel linux linux-firmware
genfstab -U /mnt >> /mnt/etc/fstab
arch-chroot /mnt
```

### Step 4
```bash
pacman -S nano
nano /etc/locale.gen
```

delete `#` before `en_US.UTF8 UTF-8`

```bash
locale-gen
echo LANG=en_US.UTF-8 > /etc/locale.conf
```

My local timezone is `Asia/Shanghai`

```bash
ln -sf /usr/share/zoneinfo/Asia/Shanghai /etc/localtime
hwclock --systohc --utc
```

Install grub

```bash
pacman -S dosfstools grub efibootmgr os-prober
grub-install --target=x86_64-efi --efi-directory=/boot/EFI --recheck
grub-mkconfig -o /boot/grub/grub.cfg
```

create a new user
```bash
useradd -m -g users -s /bin/bash YOURS_USERNAME
passwd YOURS_USERNAME
```

add to `/etc/sudoers`

```bash
YOURS_USERNAME ALL=(ALL) ALL
```

### Step 5
Install deepin desktop environment

```bash
pacman -S dhcpcd xf86-video-fbdev xorg deepin deepin-extra networkmanager
systemctl enable dhcpcd
systemctl enable networkmanager
sudo systemctl enable lightdm
```

Configure `lightdm`

```bash
sudo nano /etc/lightdm/lightdm.conf
```

below `[Seat:*]`

delete `#` and edit `greeter-session`

```
[Seat:*]
...
greeter-session=lightdm-deepin-greeter
```

edit `~/.xinitrc`, write

```bash
exec startdde
```

reboot

@bengotow

**Comment by CodeMouse92** [comment]
It appears that this issue has been resolved (#1386), so we are closing it. **If the problem occurs again, please create a new thread in the appropriate category on our [Discourse](https://community.getmailspring.com).**

We are in the process of migrating issues to Discourse, which can better facilitate discussion and discovery, and so GitHub Issues can focus on issues that are confirmed and slated for resolution in the near term. [Learn more about the changes here.](https://community.getmailspring.com/t/mailspring-community-transition/38)

We hope to see you there soon!

-The Mailspring Team
